java/vector/src/test/java/org/apache/arrow/vector/TestDictionaryVector.java

```diff
@@ -856,15 +855,15 @@ public void testEncodeStructSubFieldWithCertainColumns() {
         assertEquals(StructVector.class, encoded.getClass());
 
         assertEquals(5, encoded.getValueCount());
-        Object[] realValue1 = convertMapValuesToArray((JsonStringHashMap) encoded.getObject(0));
```

Comment:
this is just to to remove redundancy?

Reply:
Yes. Here a Map should be enough, and our revised method returns a Map.